lib/Controller/Region.php

```diff
@@ -666,6 +666,9 @@ public function preview($regionId)
             // No media to preview
             $this->getState()->extra['empty'] = true;
             $this->getState()->extra['text'] = __('Empty Region');
+        } catch (InvalidArgumentException $e) {
+            $this->getState()->extra['empty'] = true;
+            $this->getState()->extra['text'] = __('Please select a Playlist');
```

Comment:
Because this might be triggered by other things in the future (potentially) can we say "Please correct the error with this Widget" ?